python/revng/scripting/local_daemon_profile.py

```diff
@@ -0,0 +1,73 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import os
+import shutil
+import subprocess
+from signal import SIGINT, SIGKILL
+from time import sleep
+from random import randint
+
+import requests
+
+from revng.scripting.projects import DaemonProject
+
+
+class LocalDaemonProcess():
+    def __init__(self, resume_dir: str = "", revng_executable: str = "") -> None:
+        self.port: int = randint(49152, 65535)
```

Comment:
This is fragile, use [this](https://github.com/revng/revng/blob/develop/share/revng/test/tests/storage.sh#L11) to get a free port